File: AppSeleccionMateria/SistemaDeSeleccionMateria/SistemaDeSeleccionMateria/Modelos/Conexion.vb
Imports MySql.Data.Entity
Imports MySql.Data.MySqlClient
Public Class Conexion

    Private Sub Conexion()

    End Sub

    Private Shared conector As MySqlConnection

    Public Shared Function Conectar() As MySqlConnection

        Dim servidor As String = "127.0.0.1"
        Dim basedatos As String = "sistema_estudiantes"
        Dim usuario As String = "root"
        Dim contrasenia As String = ""

        Try
            conector = New MySqlConnection("Server=" + servidor + ";Database=" + basedatos + ";User=" + usuario + ";Pwd=" + contrasenia)
            conector.Open()
        Catch ex As Exception
            MessageBox.Show("No se pudo conectar" + ex.StackTrace)
        End Try

        Return conector
    End Function

    Public Shared Sub CerrarConexion()
        conector.Close()
    End Sub


End Class

File: Lavanderia0.1/Lavanderia0.1/Modelo/Conexion.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using MySql.Data.MySqlClient;
using System.Data;

namespace Lavanderia0._1.Modelo
{
    class Conexion
    {
        // ===============================================================================\\
        // Esta linea de codigo representa el metodo conectar() el cual nos va  a permitir 
        // Abrir una conexion al servidor de mysql localhost y a la base de datos BDLavanderia
        // ===============================================================================\\

        public static MySqlConnection conectar()
        {
            string servidor = "127.0.0.1";
            string bd = "dblavanderia";
            string usuario = "root";
            string clave = "";

            MySqlConnection cn = new MySqlConnection("Server=" + servidor + ";Database=" + bd + ";User=" + usuario + ";Pwd=" + clave);
            return cn;
        }
    }
}
